pytest_mh/_private/fixtures.py

```diff
@@ -80,6 +80,11 @@ def __init__(
         Multihost configuration.
         """
 
+        self.topology = topology
```

Comment:
Please, assign type. `self.topology: Topology = topology`

Reply:
IDE will figure it out, but sphinx need it to show the type in rendered documentation.

Reply:
Fixed